src/logic/map.h

```diff
@@ -517,9 +527,10 @@ class Map : public ITransportCostCalculator {
 	// Visible for testing.
 	void set_size(uint32_t w, uint32_t h);
 
-	// Change the map size
-	std::map<Coords, FieldData>
-	resize(EditorGameBase& egbase, const Coords coords, int32_t w, int32_t h);
+	// Change the map size. Must not be used outside the editor.
```

Comment:
Mmh, usig this for scenarios could be _very_ interesting. e.g. reveal a Part of a Map only after some scenario Objective.
Dropping part of the Map los tto the sea ... (ok maybe I am dreaming :-)

Reply:
In the editor we have only Immovables and Bobs and no other map object. Having buildings, flags, workers etc would make handling the splitting seams extremely difficult, so we'd better not allow that…